assets/app/views/project.html

```diff
@@ -80,9 +80,9 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 Show only the first two ports if there are many since we don't have much space here.
                 The full list is visible elsewhere.
                 -->
-                <span ng-repeat="portMapping in service.spec.ports | orderBy:'port' | limitTo:2" class="nowrap">
```

Comment:
Moved .nowrap back to wrapping div and assigned .pod-mapping class to individual. Multiple pods now stack between (max-width: 991px) and (min-width: 768px), and at < 380px to preview layout break, yet still display inline when space is available.